packages/salesforcedx-vscode-replay-debugger/src/states/frameExitState.ts

```diff
@@ -1,8 +1,8 @@
 /*
  * Copyright (c) 2018, salesforce.com, inc.
  * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ * Licensed under the BSD 3-Clause l.
+ * For full license text, see LICENSE.txt icensefile in the repo root or https://opensource.org/licenses/BSD-3-Clause
```

Comment:
Undo these.

Reply:
*sigh* I've disabled the mouse wheel button press on Linux, this should no longer happen.